### PR TITLE
Fix Qt-header include style

### PR DESCRIPTION
**Pull request by martyngigg**
Description of work.

Removes all instances of `#include <QtGui>` and `#include <QtCore>` - many files required additional includes after removing these. This also fixes up files using non-standard styles such as `#include <qwidget.h>`. 

**To test:**

Only include files have changed. A code review should be sufficient

*No issue number*

**Release Notes** 

*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- [ ] Is the code of an acceptable quality?
- [ ] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)?
- [ ] Are the unit tests small and test the class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- [ ] Do changes function as described? Add comments below that describe the tests performed?
- [ ] Do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Review by peterfpeterson — APPROVED** [review]
None of these changes are controversial.

File: qt/widgets/common/src/MantidWSIndexDialog.cpp [hunk 1]
```diff
@@ -457,7 +457,7 @@ void MantidWSIndexWidget::initWorkspaceBox() {
   m_wsBox = new QVBoxLayout;
   const QString wsIndices = m_wsIndexIntervals.toQString();
   const QString label = "Enter Workspace Indices: " + wsIndices;
-  m_wsMessage = new QLabel(tr(label.toAscii().constData()));
+  m_wsMessage = new QLabel(tr(label.toLatin1().constData()));
```
Comment (quantumsteve) [inline qt/widgets/common/src/MantidWSIndexDialog.cpp]: I think `label.toLatin1().constData()` should be replaced with `qPrintable(label)`

http://doc.qt.io/qt-5/qtglobal.html#qPrintable

**Review by quantumsteve — APPROVED** [review]
See comments for two things that caught my attention. Since the risk of merge conflicts is high, this could be done in a separate PR.

File: qt/paraview_ext/PVPlugins/Widgets/QtWidgets/SimpleBinInputWidget.cpp [hunk 2]
```diff
@@ -41,7 +41,7 @@ Getter for the current entry.
 @return current entry value
 */
 int SimpleBinInputWidget::getEntry(double, double) const {
-  return std::stoi(std::string(m_nBinsBox->text().toAscii()));
+  return std::stoi(std::string(m_nBinsBox->text().toLatin1()));
```
[the remark this answers is not in the document]
Comment (martyngigg) [reply qt/paraview_ext/PVPlugins/Widgets/QtWidgets/SimpleBinInputWidget.cpp]: You're right.

File: qt/widgets/common/src/MantidWSIndexDialog.cpp (hunk 1 above)
Comment (martyngigg) [reply qt/widgets/common/src/MantidWSIndexDialog.cpp]: Yep, that would be clearer. This was just a search-and-replace. I'll update these in a separate PR.

**Pull request merged**